device/celestica/x86_64-cel_e1031-r0/E1031-T48S4/port_config.ini

```diff
@@ -1,53 +1,5 @@
-# name          lanes   speed
-Ethernet0       1       1000
-Ethernet1       2       1000
-Ethernet2       3       1000
-Ethernet3       4       1000
-Ethernet4       5       1000
-Ethernet5       6       1000
-Ethernet6       7       1000
-Ethernet7       8       1000
-Ethernet8       9       1000
-Ethernet9       10      1000
-Ethernet10      11      1000
-Ethernet11      12      1000
-Ethernet12      13      1000
-Ethernet13      14      1000
-Ethernet14      15      1000
-Ethernet15      16      1000
-Ethernet16      17      1000
-Ethernet17      18      1000
-Ethernet18      19      1000
-Ethernet19      20      1000
-Ethernet20      21      1000
-Ethernet21      22      1000
-Ethernet22      23      1000
-Ethernet23      24      1000
-Ethernet24      25      1000
-Ethernet25      26      1000
-Ethernet26      27      1000
-Ethernet27      28      1000
-Ethernet28      29      1000
-Ethernet29      30      1000
-Ethernet30      31      1000
-Ethernet31      32      1000
-Ethernet32      33      1000
-Ethernet33      34      1000
-Ethernet34      35      1000
-Ethernet35      36      1000
-Ethernet36      37      1000
-Ethernet37      38      1000
-Ethernet38      39      1000
-Ethernet39      40      1000
-Ethernet40      41      1000
-Ethernet41      42      1000
-Ethernet42      43      1000
-Ethernet43      44      1000
-Ethernet44      45      1000
-Ethernet45      46      1000
-Ethernet46      47      1000
-Ethernet47      48      1000
-Ethernet48      54     10000
-Ethernet49      53     10000
-Ethernet50      56     10000
-Ethernet51      55     10000
+# name          lanes                 alias	index
+Ethernet-SFP-1       49           Eth1		14
+Ethernet-SFP-2       50           Eth2		15
+Ethernet-SFP-3       51           Eth3		16
+Ethernet-SFP-4       52           Eth4		17
```

Comment:
why do you remove all 1g ports? Can you keep the origin format? also I do not understand why you change lane information on 10G sfp ports.

Reply:
OK, i will bring 1g port back.
For the lanes, i reference from R0882-M1011-01- Rev1.0 Haliburton HW Design Spec.pdf

![image](https://user-images.githubusercontent.com/19452037/38533749-eec0d140-3ca4-11e8-9fb6-2f681399e940.png)